src/components/molecules/Table/Toolbar/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Button } from '@chakra-ui/react';
+import { FilterColumn, FilterRuleDefinition } from 'model/table/table-filter-types';
+import SearchBar from 'components/molecules/Table/SearchBar';
+import Filter from 'components/molecules/Table/Filter';
+import { Link as RouterLink } from 'react-router-dom';
+
+export interface ToolbarProps {
+  showButton?: boolean;
+  showFilter?: boolean;
+  showSearchBar?: boolean;
+  buttonText?: string;
+  buttonRoutePath?: string;
+  filterColumns?: FilterColumn[];
+  onSearch?: (value: string) => void;
+  onFilter?: (filterRules: FilterRuleDefinition[]) => void;
+}
+
+const Toolbar = ({
+  showButton,
+  showFilter,
+  showSearchBar,
+  buttonText,
+  filterColumns,
+  buttonRoutePath,
+  onSearch,
+  onFilter,
+}: ToolbarProps) => (
+  <Box mb="0.5rem" pb="0.5rem" display="flex" borderBottom="1px solid #e2e8f0" flexDirection="row">
+    {showButton && buttonRoutePath && (
```

Comment:
I will relate to the comment about `<AdvancedTable />`, i.e. what is the reason for `showButton` usage? 

If it's a temporary workaround then ok, but I also don't understand the idea of a condition based on routing path - since you don't currently rely on any provider, this routing should be a required props anyway right?

Reply:
During initial implementation (couple weeks ago) I wanted to make this components more generic, but as you already pointed - we always pass `true` as default prop.
I removed unnecessary props from those implementations as you suggested.